templates/katello.yaml.erb

```diff
@@ -7,10 +7,7 @@
   :content_types:
     :file: <%= @enable_file %>
     :yum: <%= @enable_yum %>
-    :deb: <%= @enable_deb %>
-    :puppet: <%= @enable_puppet %>
-    :docker: <%= @enable_docker %>
-    :ostree: <%= @enable_ostree %>
+    :docker: <%= @enable_container %>
```

Comment:
AFAIK content types are Pulp 2 only. With Pulp 3 they're discovered via capabilities in the Smart Proxy

Reply:
these are technically still needed as this tells katello what nav items to show, and what type of repositories can be created.  We could look into looking at the smart proxy for this, but aren't currently.

Reply:
Can we open an issue for that to track? From the installer side we want as little configuration as possible and a more dynamic application would be the result.